### PR TITLE
Markdown widget

### PR DESCRIPTION
**Pull request by groutr**
Render markdown files.

### CODE REVIEW

**Comment by ellisonbg** [comment]
Do we want to merge this before the talk? If so I can do a quick review now...

File: src/markdownwidget/theme.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/*-----------------------------------------------------------------------------
+| Copyright (c) Jupyter Development Team.
+| Distributed under the terms of the Modified BSD License.
+|----------------------------------------------------------------------------*/
+.jp-EditorWidget {
```
Comment (sccolbert) [inline src/markdownwidget/theme.css]: This selector will apply the border to all editor widgets, not just the markdown widgets. Is that what you're wanting, or should this be a more specific selector which only targets the markdown widgets?

**Comment by ellisonbg** [comment]
Code looks good! Amazing how simple it is...

**Comment by blink1073** [comment]
@groutr, we've made some changes to the handling of markdown, @afshin will send a PR against yours.

**Comment by ellisonbg** [comment]
Also, checked are failing here...

**Comment by groutr** [comment]
@ellisonbg Thanks.  I'm not sure why it's failing on travis.  It looks like maybe `sanitizer` isn't installed?  There is likely something I'm not understanding about typescript.  The import block was copied from `renderers/index.js`

**Comment by blink1073** [comment]
@groutr, we switched away from `sanitizer` to `santize-hmtl`, but you still had `sanitizer` installed locally.

**Comment by afshin** [comment]
@groutr I'm working on a PR to your branch now, shouldn't be too long.

**Comment by ellisonbg** [comment]
Continued in #384

**Pull request closed without merging**